autoware_msg_bag_converter/package.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>autoware_msg_bag_converter</name>
+  <version>0.1.0</version>
+  <description>TODO: Package description</description>
```

Comment:
You should probably make a change because there is still a TODO left.

Reply:
I fixed with the following commit hash
95a227027c351b1a8980b6f40471158ddcb5bb40